builds/make.host.frontier

```diff
@@ -8,11 +8,12 @@ GPUCXX           ?= hipcc
 CFLAGS_DEBUG      = -g -O0
 CFLAGS_OPTIMIZE   = -g -O2
 
-CXXFLAGS_DEBUG    = -g -O0 -std=c++11
-CXXFLAGS_OPTIMIZE = -Ofast -std=c++11
+CXXFLAGS_DEBUG    = -g -O0 -std=c++17
+CXXFLAGS_OPTIMIZE = -g -Ofast -std=c++17 -Wno-unused-result
 
-GPUFLAGS          = --offload-arch=gfx90a
-HIPCONFIG	  = $(shell hipconfig -C)
+GPUFLAGS          = --offload-arch=gfx90a -Wno-unused-result 
+HIPCONFIG	  = -I$(ROCM_PATH)/include $(shell hipconfig -C) # workaround for Rocm 5.2 warnings
```

Comment:
`hipconfig` needs to be fixed for Rocm 5.2. I plan to submit an issue to AMD on this. Until it's fixed, we need the extra `-I` path.